internal/feedback/feedback.go

```diff
@@ -104,6 +104,9 @@ func GetLockingFeedback(version, revision, depType, projectPath string) string {
 	}
 
 	if depType == DepTypeImported {
+		if version == "" {
+			version = "*"
+		}
```

Comment:
Also, add a test for this change in [`TestFeedback_LockedProject`](https://github.com/golang/dep/blob/master/internal/feedback/feedback_test.go#L54) ?